docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,190 @@
+|`tags` |String[] |An array of strings containing words and phrases to help categorize
+exception items. Tags can also be used to define the {endpoint-artifact-name-single} as either globally applicable to all policies or assigned to one or more policies (per policy). See xref:exception-item-scope-assignment-tags[] for more details. |No
+|`type` |String a|Must be `simple`. |Yes
+|`_version` |String a|The version id, normally returned by the API when the item was retrieved. Use it ensure updates are done against the latest version. |No
+
+|==============================================
+// end::update-exception-item-request-body[]
+
+
+
+// tag::api-id-or-itemid-query-params[]
+The URL query must include one of the following:
+
+* `id` - the `id` of the item, or
+* `item_id` -  the `item_id` of the item
+
+In addition to the above, `namespace_type` URL query parameter is also required with a value of `agnostic`
+// end::api-id-or-itemid-query-params[]
+
+
+// tag::find-exception-items-query-params[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
+|`page` |Integer |The page number to return. |No
+|`per_page` |Integer |The number of items to return per page. |No
+|`sort_field` |String |Determines which field is used to sort the results. |No
+|`sort_order` |String a|Determines the sort order, which can be `desc` or `asc`. |No
+|`filter` |String |A {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)] string to filter the results down. |No
+
+|==============================================
+// end::find-exception-items-query-params[]
+
+
+
+// tag::endpoint-exceptions-value-types-info-section[]
+==== Value types
+
+The following describes the types that can be defined when using the xref:create-item-api[create] or xref:update-item-api[update] {endpoint-artifact-name} APIs:
+
+[#exception-item-comment-schema]
+===== `comment` object schema
+
+Comments are JSON objects containing the following structure:
+
+[source,json]
+---------
+{
+    "comment": "some comment here"
+}
+---------
+
+When used with the update API, existing comments can be updated by using their associated `id`, while any comment without the `id` attribute will be added as a new comment:
+
+[source,json]
+---------
+{
+    "comment": "some comment here - updated",
+    "id": "1078cf59-5893-4143-acf7-40a40af16bee"
+}
+---------
+
+[#exception-item-os-types]
+===== `os_types` values
+
+ifeval::["{endpoint-artifact-list-id}" == "endpoint_host_isolation_exceptions"]
+All of the following valid OS types should be used when creating {endpoint-artifact-name}:
+endif::[]
+ifeval::["{endpoint-artifact-list-id}" != "endpoint_host_isolation_exceptions"]
+The following are the valid OS types that can be used when creating {endpoint-artifact-name}:
+endif::[]
```

Comment:
all OSs types are required for host isolation exceptions

Reply:
This is probably not needed here, since this section is just describing the general structure/values available for the fields that need expanded docs